### DISCUSSION TITLE
Options order (select for answer )

### DISCUSSION THREAD

**Discussion by afnanmr** [Budibase/budibase#4458] [Help]
![Screen Shot 2022-02-13 at 1 21 41 PM](https://user-images.githubusercontent.com/93825965/153749181-59a64ff5-d697-4ba3-a70f-e0086402faea.png)

Hi Guys , 
i have created a form on my app and found an issue that i can't arrange  the selection options for the answer, to be make sense for the user 
the budibased is ordering by alphabet sorting even though  i arranged manually on database

**Comment by SMetriq** [comment]
Hello, i'm not yet at this level but this is not possible to use a table with an id and order on it ?

**Comment by mjashanks** [comment]
Hey @afnanmr 

There are a few different ways in which you can load options:

![image](https://user-images.githubusercontent.com/3524181/153857282-c67fffc9-35cf-4c37-b3e1-6aaafba62117.png)

1. Schema - options are loaded from the table definition. The only way to order these is to name them alphabetically
e.g.
`1 - Entry Level`
`2- Associate` etc
2. Custom - options are declared inside the select. Again you need to name them alphabetically
3. Data provider. In this case, you load options from a `Data Provider`. In this case, you can declare a Sort column in your dataprovider. I.e. you could add an "Order" column to your table, that defines the order... I explain how to set this all up here...
https://github.com/Budibase/budibase/discussions/4418#discussioncomment-2156404
